src/Types/ApprenticeshipCreatedEvent.cs

```diff
@@ -1,24 +1,11 @@
 ﻿namespace SFA.DAS.Apprenticeships.Types;
 
-public class ApprenticeshipCreatedEvent
+public class ApprenticeshipCreatedEvent : EpisodeUpdatedEvent
```

Comment:
In earnings I handle this event separately to the episode updates (price change, start date change). I can see why you have added this inheritance in because we need the episode object and it's valid from that standpoint, but it's not really an updated event in the same way. I am maybe thinking that it might be cleaner to leave this type as inherited only by the change events and just putting an Episode property on this one?